res/res_pjsip_endpoint_identifier_ip.c

```diff
@@ -305,10 +305,37 @@ static struct ast_sip_endpoint *ip_identify(pjsip_rx_data *rdata)
 	return common_identify(ip_identify_match_check, &addr);
 }
 
+static struct ast_sip_endpoint *transport_identify(pjsip_rx_data *rdata)
+{
+	char buffer[PJ_INET6_ADDRSTRLEN];
+	struct ast_sockaddr addr = { { 0, } };
+	union pj_sockaddr sock = rdata->tp_info.transport->local_addr;
+	pj_status_t status;
+
+	if (sock.addr.sa_family == PJ_AF_INET6) {
```

Comment:
Just use pj_sockaddr_print instead of differentiating based on IPv4/IPv6.

Reply:
Unfortunately pj_sockaddr_print expects the first argument to be a const pj_sockaddr_t, however transport->local_addr is a union pj_sockaddr _(without the _t suffix)_. So I would like to leave this as it is.